setup.md

```diff
@@ -51,7 +52,7 @@ The installer for Anaconda can be found on [this page](https://www.anaconda.com/
 
 Throughout the rest of this set-up, we will assume that you are using the `conda` package manager.
 
-Please set up your python environment at
+Please set-up your python environment
```

Comment:
Thank you for removing the incorrect word "at". However "set up" should not have a hyphen here. "Set up" as a verb contains no hypen. https://www.merriam-webster.com/dictionary/setup#note-1